deploy/sre-prometheus/100-sre-infra-node-imbalance.PrometheusRule.yaml

```
@@ -11,7 +11,7 @@ spec:
   - name: sre-infra-node-imbalance
     rules:
     - alert: KubeInfraNodeImbalanceSRE
-      expr: count(count by (node) (kube_pod_info{namespace="openshift-monitoring",pod=~"prometheus-k8s-.*"})) != count(cluster:infra_nodes)
+      expr: count(count by (node) (kube_pod_info{namespace="openshift-monitoring",pod=~"prometheus-k8s-.*"})) < 2
```

Wouldn't this be better as something like this:
```suggestion
      expr: count by (node) (kube_pod_info{namespace="openshift-monitoring",pod=~"prometheus-k8s-.*"}) > 1
```

If the count per node is greater than 1, then there is an imbalance